anet.yml

```diff
@@ -155,15 +155,16 @@ dictionary:
   POAM_LONG_NAME: Plan of Action and Milestones / Pillars
   POAM_SHORT_NAME: PoAM
   NAV_BAR_ALL_ADVISOR_ORGS: All EFs / AOs
-  PRINCIPAL_ORG_LABEL_LONGNAME: Official Organization Name
   PRINCIPAL_ORG_PLACEHOLDER_LONGNAME: e.g. Afghan Ministry of Defense
-  ADVISOR_ORG_LABEL_LONGNAME: Description
   ADVISOR_ORG_PLACEHOLDER_LONGNAME: e.g. Force Sustainment
-  PRINCIPAL_ORG_LABEL_IDENTIFICATIONCODE: Identification Code
   PRINCIPAL_ORG_PLACEHOLDER_IDENTIFICATIONCODE: optional
-  ADVISOR_ORG_LABEL_IDENTIFICATIONCODE: UIC
   ADVISOR_ORG_PLACEHOLDER_IDENTIFICATIONCODE: the six character code
   SUPPORT_EMAIL_ADDR: support@example.com
+  field_labels:
```

Comment:
The labels and placeholders are now too far apart; maybe have something like:
```
  fields:
    labels:
      ADVISOR_ORG_LONGNAME: fred
      …
    placeholders:
      ADVISOR_ORG_LONGNAME: ask fred
      …
```
?

Reply:
or maybe even:
```
  fields:
      ADVISOR_ORG_LONGNAME: 
         label: fred
         placeholder: ask fred
      …
```

Reply:
@gjvoosten shall I go with @VassilIordanov 's format?

Reply:
I have a slight dislike of the alternative (that I also considered, but rejected), as it is too repetetive (each key has subkeys `label` and `placeholder`).

Reply:
doesn't matter to me that much. I would go with the former if these are the only two parameters we are planning to use. I would go with the latter if we plan to evolve it into something like:
```
  fields:
      TaskCustom: 
         label: Task status
         placeholder: ask fred
         Enum:
            OPEN: Open
            CLOSED: Closed
         Hint: Task status is blah blah blah
```

Reply:
I think if we keep all data pertaining to the field together, it is indeed more verbose, but there is less room for mistakes. In the current structure, we already have fields with different requirements, such as:
```  fields:
      ADVISOR_ORG_LONGNAME: 
         label: fred
         placeholder: ask fred
      TASK_CUSTOM_FIELD:
         label: xyz
         placeholder: xyz
      TASK_CUSTOM_ENUM: 
         label: Task status
         Enum:
             OPEN: Open
             CLOSED: Closed
    ....```

Reply:
Then let's go all the way.